Source/Charts/Highlight/Highlight.swift

```diff
@@ -176,81 +176,25 @@ open class Highlight: NSObject
     
     open override func isEqual(_ object: Any?) -> Bool
     {
-        if object == nil
-        {
-            return false
-        }
-        
-        if !(object! as AnyObject).isKind(of: type(of: self))
-        {
-            return false
-        }
-        
-        if (object! as AnyObject).x != _x
-        {
-            return false
-        }
-        
-        if (object! as AnyObject).y != _y
-        {
-            return false
-        }
-        
-        if (object! as AnyObject).dataIndex != dataIndex
-        {
-            return false
-        }
-        
-        if (object! as AnyObject).dataSetIndex != _dataSetIndex
-        {
-            return false
-        }
-        
-        if (object! as AnyObject).stackIndex != _stackIndex
-        {
-            return false
-        }
-        
-        return true
+        guard let object = object as? Highlight else { return false }
+        return self == object
     }
 }
 
-func ==(lhs: Highlight, rhs: Highlight) -> Bool
-{
-    if lhs === rhs
+// MARK: Equatable
+extension Highlight /*: Equatable*/ {
```

Comment:
Why is equatable commented out?

Reply:
Because `NSObject` adopts `Equatable` and you can't declare conformance twice. When Obj-C support is dropped, then we will need to uncomment it.